README.md

```diff
@@ -30,6 +30,39 @@ Note: The PoC application shown here, is ideal for demonstration purposes only.
 
 ---
 
+## E2E Testing
+
+For e2e testing we are using cypress and **docker image** - `image: "cypress/included:3.2.0"` which comes with preinstalled cypress. Cypress config and tests can be found in `./cypress` folder. Test flows or Cypress specs are located in `/cypress/integration` folder. Each spec should represent a flow e.g.
```

Comment:
_preinstalled Cypress_ ?

Reply:
updated this element as well, we are no longer using a docker image